packages/discord.js/typings/index.test-d.ts

```diff
@@ -2593,3 +2594,16 @@ declare const poll: Poll;
 expectType<Collection<Snowflake, StickerPack>>(await client.fetchStickerPacks());
 expectType<Collection<Snowflake, StickerPack>>(await client.fetchStickerPacks({}));
 expectType<StickerPack>(await client.fetchStickerPacks({ packId: snowflake }));
+
+client.on('interactionCreate', interaction => {
+  if (!interaction.channel) {
+    return;
+  }
+
+  // @ts-expect-error
+  interaction.channel.send();
+
+  if (interaction.channel.isSendable()) {
+    expectType<SendableTextBasedChannels>(interaction.channel);
+  }
```

Comment:
To not only check typeguard with type but also actual purpose, maybe add: 
```suggestion
    interaction.channel.send()
  }
```